### PR TITLE
docs: add OCCLUM_RELEASE_ENCLAVE env to explicitly set enclave mode

### PR DESCRIPTION
**Pull request by hustliyilin**
Fixes: #507
Signed-off-by: Yilin Li <YiLin.Li@linux.alibaba.com>

### CODE REVIEW

File: docs/running_inclavare_containers_with_pouch_and_occlum.md [hunk 1]
```diff
@@ -17,14 +17,21 @@ Please refer to [this guide](https://github.com/alibaba/inclavare-containers/tre
 
 # Running Occlum container image
 
-Please refer to [this guide](https://github.com/occlum/occlum/blob/master/docs/rune_quick_start.md) to build your Occlum container image. Then run pouch with Occlum container images refer to
+Please refer to [this guide](https://github.com/occlum/occlum/blob/master/docs/rune_quick_start.md) to build your Occlum container image. 
+
+You can use `OCCLUM_RELEASE_ENCLAVE` env to set your enclave type:
```
Comment (jiazhang0) [inline docs/running_inclavare_containers_with_pouch_and_occlum.md:22]: change to: use the environment variable `OCCLUM_RELEASE_ENCLAVE` to set your enclave type:

File: docs/running_inclavare_containers_with_pouch_and_occlum.md [hunk 2]
```diff
@@ -17,14 +17,21 @@ Please refer to [this guide](https://github.com/alibaba/inclavare-containers/tre
...
+- `debug enclave`: `OCCLUM_RELEASE_ENCLAVE=0`,
+- `product enclave`: `OCCLUM_RELEASE_ENCLAVE=1`
```
Comment (jiazhang0) [inline docs/running_inclavare_containers_with_pouch_and_occlum.md:24]: - `OCCLUM_RELEASE_ENCLAVE=0`: debug enclave
- `OCCLUM_RELEASE_ENCLAVE=1`: product enclave

File: rune/README.md [hunk 3]
```diff
@@ -87,15 +87,28 @@ rune spec
...
   "process": {
       "args": [
-          "/bin/hello_world"
+	   "/bin/hello_world"
       ],
   }
 ```
 
-and then configure enclave runtime as following:
+and then set the `OCCLUM_RELEASE_ENCLAVE` process.env to set whether your occlum enclave is `product enclave` or `debug enclave`. the default enclave mode is `debug enclave` with `OCCLUM_RELEASE_ENCLAVE=0`. If you have a `product enclave` of `occlum`, you need to set `"OCCLUM_RELEASE_ENCLAVE=1`.
```
Comment (jiazhang0) [inline rune/README.md:99]: to set your occlum enclave is either product enclave or debug enclave.

Comment (jiazhang0) [inline rune/README.md:99]: Change to : 

```
and then set the `OCCLUM_RELEASE_ENCLAVE` process.env to configure the enclave mode. The default mode is debug. Please specify `OCCLUM_RELEASE_ENCLAVE=1` if using a product enclave.
```

**Review by jiazhang0 — CHANGES_REQUESTED** [review]

File: docs/running_inclavare_containers_with_pouch_and_occlum.md (hunk 2 above)
Comment (hustliyilin) [reply docs/running_inclavare_containers_with_pouch_and_occlum.md]: > * `OCCLUM_RELEASE_ENCLAVE=0`: debug enclave
> * `OCCLUM_RELEASE_ENCLAVE=1`: product enclave

Done.

File: docs/running_inclavare_containers_with_pouch_and_occlum.md (hunk 1 above)
Comment (hustliyilin) [reply docs/running_inclavare_containers_with_pouch_and_occlum.md]: > change to: use the environment variable `OCCLUM_RELEASE_ENCLAVE` to set your enclave type:

Done

**Review by jiazhang0 — APPROVED** [review]

**Pull request merged**